test/executor.test.js

```diff
@@ -271,6 +271,35 @@ describe('executor', function() {
         done();
       });
     });
+
+    describe ('for mixins', function() {
+      it('defines `mixins/*` files', function() {
+        boot.execute(app, simpleAppInstructions());
+
+        var modelBuilder = app.loopback.modelBuilder;
+        var registry = modelBuilder.mixins.mixins;
+
+        expect(registry).to.have.property('Example');
+        expect(registry).to.have.property('Timestamping');
+
+        var meta = modelBuilder.mixins.meta;
+        expect(meta).to.have.property('Example');
+        expect(meta).to.have.property('Timestamping');
+      });
+
+      it('supports mixin as model', function() {
+        boot.execute(app, simpleAppInstructions());
+
+        var modelBuilder = app.loopback.modelBuilder;
+        var meta = modelBuilder.mixins.meta;
+        var json = require('./fixtures/simple-app/mixins/time-stamps.json');
+
+        expect(meta.Timestamping.name).to.equal(json.name);
+        expect(meta.Timestamping).to.have.property('filepath');
+        expect(meta.Timestamping.description).to.equal(json.description);
+        expect(meta.Timestamping.options).to.eql(json.options);
+      });
+    });
```

Comment:
When looking back, writing unit-tests via `simpleAppInstructions()` was a mistake. Please write the new tests using the approach where all files are created via `appdir.writeFileSync` and friends, as you already do in compiler tests. Other executor tests are already following this pattern, see for example ["it disables component when configuration is not set"](https://github.com/PradnyaBaviskar/loopback-boot/blob/9704c70f465947465d9bf5e5976a036cebfae072/test/executor.test.js#L506-L519).